components/docs-chef-io/content/automate/ha_upgrade_introduction.md

```diff
@@ -12,4 +12,40 @@ gh_repo = "automate"
     weight = 200
 +++
 
-TO BE ADDED
\ No newline at end of file
+## What is Upgradation?
+
+Upgradation is a process of replacing older version with improved newer version with some new features or/and fixes for bugs that has been noticed in older versions. Use the --major flag to upgrade from the latest to a major version. This section will talk about the version upgrade of Chef Automate with its possible scenarios.
+
+{{< note >}} Unlike some of the other chef services, Automate HA won't upgrade from one minor version to another. This needs to be done manually by running command. {{< /note >}}
+
+## Release Channels
+
+The Chef Automate upgrade process makes use of **release channels** to allow greater control over the automatic upgrades applied to your system. Chef Automate pulls the latest release within a specified release channel. The default channel is the `current` channel.
```

Comment:
REPLACE IT WITH: The Chef Automate upgrade process uses **release channels** to allow greater control over the automatic upgrades applied to your system. Chef Automate pulls the latest release within a specified release channel. The default channel is the `current` channel.

Reply:
Done